contracts/mixnet/src/queries.rs

```diff
@@ -0,0 +1,187 @@
+// settings for pagination
+use crate::state::MixNodeBond;
+use crate::state::PREFIX_MIXNODES;
+use cosmwasm_std::Deps;
+use cosmwasm_std::HumanAddr;
+use cosmwasm_std::Order;
+use cosmwasm_std::StdResult;
+use cosmwasm_storage::bucket_read;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+const MAX_LIMIT: u32 = 30;
```

Comment:
Isn't the default limit of 30 kinda too small considering we are going to have over 1,500 mixnodes when this is first deployed?